htsget-search/src/htsget/mod.rs

```diff
@@ -92,12 +92,12 @@ pub struct Query {
   pub class: Class,
   /// Reference name
   pub reference_name: Option<String>,
-  /// sequence start position (1-based)
+  /// sequence start position (0-based, inclusive, exclusive)
```

Comment:
What do you mean by "inclusive, exclusive"?

Reply:
It is supposed to mean the start of the range is inclusive, but the end is exclusive, but after a second look, I agree the meaning isn't obvious.

Probably better to leave just inclusive above the start and exclusive above the end. I'm also open to other suggestions!

Reply:
Ah, gotcha, how about just using the typical mathematical notation for it?: https://stackoverflow.com/a/4396303/457116

Reply:
Yeah, that seems better 👍🏻